src/components/EntitlementsAgreementsList/EntitlementAgreementsList.js

```diff
@@ -37,12 +43,25 @@ const EntitlementAgreementsList = (
   };
 
   const columnWidths = {
+    name: { min: 200, max: 400 },
     startDate: 120,
     endDate: 120,
   };
 
   const formatter = {
-    name: e => <div data-test-agreement-name><Link to={urls.agreementView(e?.owner?.id)}>{e?.owner.name}</Link></div>,
+    name: e => {
+      const iconKey = e?.owner.agreementStatus?.value === statuses.CLOSED ? 'closedAgreement' : 'app';
+      return (
+        <AppIcon
+          app="agreements"
+          iconAlignment="baseline"
+          iconKey={iconKey}
+          size="small"
+        >
+          <div data-test-agreement-name><Link to={urls.agreementView(e?.owner?.id)}>{e?.owner.name}</Link></div>
```

Comment:
`e?.owner.name` should actually be `e?.owner?.name`. But, this also tells that the name could be `undefined` here, so there needs to be a fallback. If not, `undefined` would be displayed on the UI

Reply:
`data-test-agreement-name` is added here but I do not see any related tests?

Reply:
I didn't touch this piece of code, I just wrapped it into `AppIcon`